### DISCUSSION TITLE
Run lively application without installing it.

### DISCUSSION THREAD

**Discussion by ashutoshgs** [rocksdanister/lively#2064] [Q&A]
I want to install this on my company laptop. As any company would, it doesn't allow me to run a .exe file. Is there any other way I can run this application without running the .exe file. Any help of information on this is highly appreciated

**Comment by rocksdanister** [comment]
This is something you need to talk with your company IT about, Lively is a consumer product and is not meant for work machines.

As for question about running Lively without installing, simply copying the install folder from one system to another should work (provided dotnet runtime and visual c++ dependencies are installed) but data files will be created in appdata folder.
